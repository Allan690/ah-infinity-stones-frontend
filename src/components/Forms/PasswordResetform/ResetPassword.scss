.new_password_label{
    margin-right: 28px;
}

.repeat_password_label{
    margin-right: 8px;
}

.input{
    padding: 7px;
    border-radius: 5px;
    width: 250px;
}

.button-success{
    // padding: 8px 1px;
    margin-left: 25px;
    //  border-radius: 5px;
    color: white;
    background-color: #86c232;
    //  width: 300px;
    margin-left: 130px;
    padding: 7px;
    border-radius: 5px;
    width: 270px;
}

/* The message box is shown when the user clicks on the password field */
.message {
    display:none;
    background: #f1f1f1;
    color: #000;
    position: relative;
    padding: 20px;
    margin-top: 10px;
  }

  #message p {
    padding: 10px 35px;
    font-size: 18px;
  }

  .alert {
    color: #4F8A10;
    font-family: 'Courier New', Courier, monospace;
}
