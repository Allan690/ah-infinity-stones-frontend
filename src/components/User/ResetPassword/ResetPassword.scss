$color-black:black;
p { color: $color-black; }

.form_container{
    margin: 80px auto;
    padding: 10px 20px 70px 20px;
    border: 1px solid #e6e6e6;
    border-radius: 5px;
    width: 460px;
    h1{
        color: #86c232;
    }
}

body{
    color: black!important;
}

.button-success.disabled {
    cursor: not-allowed
}
