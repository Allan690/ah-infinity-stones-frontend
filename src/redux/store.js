/* eslint no-underscore-dangle: 0 */
/* eslint-disable no-undef */
import { applyMiddleware, createStore, compose } from 'redux';
import createSagaMiddleware from 'redux-saga';
import allReducers from './reducers';
import rootSaga from './sagas';

// only activate redux devtools on the browser otherwise just compose
const composer = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
const composeEnhancers = composer;

const middleware = createSagaMiddleware();
const store = createStore(
  allReducers,
  composeEnhancers(applyMiddleware(middleware)),
);

export default store;

middleware.run(rootSaga);
