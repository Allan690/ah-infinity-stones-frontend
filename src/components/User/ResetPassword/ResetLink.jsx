import React, { Component } from 'react';
import './ResetPassword.scss';
import ResetLinkForm from '../../Forms/PasswordResetform/Resetlink';

class ResetLink extends Component {
  render() {
    return (
      <div className="form_container">
        <h1>Reset Password Email</h1>
        <hr />
        <h5>
          <p>
          </p>
        </h5>

        <ResetLinkForm />
      </div>
    );
  }
}

export default ResetLink;
