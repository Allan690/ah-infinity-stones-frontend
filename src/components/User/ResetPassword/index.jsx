import React, { Component } from 'react';
import './ResetPassword.scss';
import ResetPasswordForm from '../../Forms/PasswordResetform/index';

class ResetPassword extends Component {
  render() {
    return (
      <div className="form_container">
        <h1>Reset Password</h1>
        <hr />
        <h5>
          <p>

          </p>

        </h5>

        <ResetPasswordForm />


      </div>
    );
  }
}


export default ResetPassword;
