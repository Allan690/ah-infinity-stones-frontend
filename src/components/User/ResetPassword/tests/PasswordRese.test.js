import React from 'react';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import ResetPassword from '../index';
import store from '../../../../redux/store';

describe('ResetPassword form', () => {
  let passwordkContainer;
  let reduxpassword;
  beforeEach(() => {
    passwordkContainer = shallow(<ResetPassword className="form_container" />);
    reduxpassword = mount(
      <Provider store={store}>
        <BrowserRouter>
          <ResetPassword />
        </BrowserRouter>
      </Provider>,
    );
  });
  it('form renders successfully', () => {
    expect(passwordkContainer.exists()).toEqual(true);
  });
  it('Should match snapshots', () => {
    expect(passwordkContainer).toMatchSnapshot();
  });
  it('Reset link should render with redux store', () => {
    expect(reduxpassword.exists()).toEqual(true);
  });
  it('renders props correctly', () => {
    const Reset = shallow(<ResetPassword name="ResetPassword" />);
    expect(Reset.instance().props.name).toBe('ResetPassword');
  });
});
