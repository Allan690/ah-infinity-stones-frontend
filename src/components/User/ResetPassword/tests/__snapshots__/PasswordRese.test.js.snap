// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResetPassword form Should match snapshots 1`] = `
<div
  className="form_container"
>
  <h1>
    Reset Password
  </h1>
  <hr />
  <h5>
    <p />
  </h5>
  <Connect(ResetPasswordForm) />
</div>
`;
